KeenClient.podspec

```diff
@@ -21,8 +21,8 @@ Pod::Spec.new do |spec|
   spec.source       = { :git => 'https://github.com/keenlabs/KeenClient-iOS.git', :tag => spec.version.to_s }
   spec.source_files = 'KeenClient/*.{h,m}','Library/Reachability/*.{h,m}'
   spec.public_header_files = 'KeenClient/*.h'
-  spec.frameworks   = 'CoreLocation'
   spec.requires_arc = true
+  spec.frameworks = 'SystemConfiguration', 'CoreLocation', 'CFNetwork'
```

Comment:
The concern may have actually just been about requiring users to add CFNetwork.framework to their list of link targets. This would only ever be a problem if they're manually linking a static version of the library into their projects. I think it's reasonable to assume if you're doing that with your project then having to manually specify library dependencies isn't a big deal. Most people are going to be using Carthage or CocoaPods. Carthage only ever builds as a framework, and CocoaPods knows to look at the podspec for dependencies if you choose to build as a static library.